cozepy/websockets/ws.py

```
@@ -184,6 +185,40 @@
     voice_id: Optional[str] = None
 
 
+class WebsocketsEventFactory(object):
+    def __init__(self, event_type_to_class: Dict[str, Type[WebsocketsEvent]]):
+        self._event_type_to_class = event_type_to_class
+
+    def get_data_type(cls, event_class: WebsocketsEvent) -> Optional[Type[BaseModel]]:
```

Pylint (3.3.7)

[error] Method 'get_data_type' should have "self" as first argument

(E0213)